Update framework components readme - Closes #3158 (#3629)
LiskHQ/lisk-sdk: framework/src/components/README.md

Find more details about the storage component in the dedicated [LIP](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0011.md).

## Configuration

Configuration options for each component are located in `framework/src/component/<component-name>/defaults/config.js`.

Each `config.js` file consists of 2 parts:

> 1. A general schema description of all available config options.

Reviewer: ```suggestion
1. JSON-schema specification for all available config options.
```

2. Default values for the available config options for this specific compoment.

> To customize the predefined defaults values, please don't change the default values in these files directly, as they will be overwritten on software updates.

Reviewer: ```suggestion
Please don't change the default values in these files directly as they will be overwritten on software updates, instead define the custom configuration options inside your blockchain application.
```

> Instead, define the custom configuration options inisde your blockchain application, and let it overwrite the default config options.

Reviewer: ```suggestion
```
